training_record/serializers.py

```diff
@@ -52,12 +53,18 @@ class ReadOnlyRecordSerializer(serializers.ModelSerializer):
     '''Indicate how to serialize Record instance for reading.'''
     status_str = serializers.CharField(source='get_status_display',
                                        read_only=True)
+    role_str = serializers.CharField(
+        source='event_coefficient.get_role_display',
+        read_only=True)
+    role = serializers.IntegerField(
```

Comment:
I think you don't need to define `role` field, it's already there, declaring it in the `fields` is enough.